### PR TITLE
Footer instagram section

### PR DESCRIPTION
**Pull request by amagdy46**
https://abdomgdy.fitness.m3ntorship.com

### CODE REVIEW

**Review by Nilegfx — APPROVED** [review]
Sorry, wrong review ..

**Review by Nilegfx — CHANGES_REQUESTED** [review]
Please check the footer blog section and be consistent with it. I think there is a good structure for class names and HTML elements there. feel free to communicate with @MoHassn in this regard

File: dist/index.html [hunk 1]
```diff
@@ -232,7 +232,27 @@ <h2>Fitness</h2>
           Blog Posts
         </div>
         <div class="footer-section__feed">
-          instagram or twitter news feed
+          <div class="row_header">
+            <p>INSTAGRAM</p>
```
Comment (MoHassn) [inline dist/index.html]: I recommend you to use `<h3> ` instead `<p>`
you could find more about here:
https://www.w3schools.com/tags/tag_p.asp
https://www.w3schools.com/tags/tag_hn.asp

Comment (amagdy46) [reply dist/index.html]: Thanks for your suggestion i edited it and commited again.

File: dist/index.html [hunk 2]
```diff
@@ -445,23 +445,28 @@ <h3>BLOGS</h3>
...
+            <img src="assests/img/instagram/insta5.jpg" alt="Insta Pic" />
+            <img src="assests/img/instagram/insta6.jpg" alt="Insta Pic" />
+          </div>
+          <div class="img_row">
+            <img src="assests/img/instagram/insta7.jpg" alt="Insta Pic" />
+            <img src="assests/img/instagram/insta8.jpg" alt="Insta Pic" />
+            <img src="assests/img/instagram/insta9.jpg" alt="Insta Pic" />
+          </div>
+          <div class="options">
+            <a>View More Photos..</a>
```
Comment (Nilegfx) [inline dist/index.html]: please add `href` attribute here to get the mouse hover pointer to work.

**Review by tramsi — APPROVED** [review]
LGTM!
Great work!!

**Review by tramsi — APPROVED** [review]
LGTM!

**Comment by tramsi** [comment]
> Please check the footer blog section and be consistent with it. I think there is a good structure for class names and HTML elements there. feel free to communicate with @MoHassn in this regard

A ticket has been made to fix these changes after merging this PR 
https://trello.com/c/WnhqkEBN/34-html-css-fix-footer-design

**Review by Nilegfx — APPROVED** [review]

**Pull request merged**